pkg/config/template.go

```diff
@@ -95,6 +95,8 @@ type configuration struct {
 		MaxOpenConnections    int32         `mapstructure:"max_open_connections"`
 		MaxIdleConnections    int32         `mapstructure:"max_idle_connections"`
 		ConnectionMaxLifetime time.Duration `mapstructure:"connection_max_lifetime"`
+		Type                  string        `mapstructure:"type"`
```

Comment:
What is `Type`?

Reply:
Same convention as in Blockstore Type
Used to define which DB implementation to use, it is in fact the name of the driver to load

Reply:
Let's document that with the possible values. Isn't it an enum?